entrypoint

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -o pipefail
+set -o errexit
+
+(cd /src/Bundle && composer install)
```

Comment:
What does it mean to use parenthesis around your commands?

Reply:
It means that it's scoped and the `cd` executed inside it does not affect following commands.

Reply:
ok then :+1: